examples/proto/gostyle/BUILD

```diff
@@ -1,8 +1,15 @@
 load("//proto:go_proto_library.bzl", "go_proto_library")
 
+genrule(
+    name = "copy",
+    srcs = ["gostyle.proto"],
+    outs = ["gostyle.gen.proto"],
+    cmd = "sed -e 's/ano.proto/any.proto/' $< > $@",
```

Comment:
I don't understand why this sed is necessary. ano.proto does not appear anywhere in gostyle.proto, and even if it did, why would we want this instead of just changing the proto?